Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/buttons/Hyperlink.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import concatClassNames from '../../../utils/class-names';
+import generateId from '../../../utils/id-generator';
+
+import styles from './Hyperlink.module.scss';
+
+interface IHyperlinkProps {
+    to: string;
+    text: string;
+    id?: string;
+    className?: string | string[];
+    isToExternal?: boolean;
+}
+
+const Hyperlink = ({
+    to,
+    text,
+    className = '',
+    id = generateId(),
+    isToExternal = false,
+ }: IHyperlinkProps ) => {
+    const linkClassName = concatClassNames(className, styles.link);
+
+    const toHref= isToExternal

```

Comment:
State

Reply:
Why state here? How a param can be changed without rerendering the whole component anyway?